jupyter/minimal/ubi8-python-3.8/start-notebook.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
This should be:

```shell
#!/usr/bin/env bash
```

See https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash